sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/credentials/ConfigurationClientCredentials.java

```diff
@@ -73,6 +73,7 @@ public URL baseUri() {
      * @param httpMethod the request HTTP method
      * @param contents the body content of the request
      * @return a flux of headers to add for authorization
+     * @throws NoSuchAlgorithmException If the SHA-256 algorithm doesn't exist.
```

Comment:
Where do you throw the NoSuchAlgorithm?

Reply:
Good point, apparently `Exception.propagate` doesn't throw the exception eagerly but bubbles it up through the reactive stream.

I will remove this.